### DISCUSSION TITLE
Lazer made me signifficantly better in just 3 days somehow

### DISCUSSION THREAD

**Discussion by SolsticeSpectrum** [ppy/osu#28087] [Ideas]
I've been into classic osu for a while, but honestly, I never seemed to get any better. I was always stuck on 1-2* maps, no matter what I tried. I even switched to using like pro skins and turned off backgrounds, thinking it might help, but no dice. And it's not just osu. I've never been able to get the hang of games like Counter Strike or chess and other skill based games either. I guess you could say I'm just not great at games in general.

Then comes lazer. I don't know if it's the cool sound effects in Argon Pro, the slick design of Argon itself, or maybe just the chill colors and little touches like the background blur, audio reactive UI elements in main menu, custom rulesets and other stuff that generally make me more happy to play the game. But whatever it is, it made a huge difference. I mean, in just three days, I went from barely managing 2* maps to taking on 3*+ maps.

I can't really explain why it works so well for me, but whatever the developers did, it's seriously working. Like, really working. Lazer has not only upped my performance but also boosted my confidence big time. So, big shoutout to ppy and all the contributors for making this game.

You might think I'm going a bit overboard with my excitement, but for someone like me who's always struggled with games, this progress is a big deal. I'm feeling pretty grateful to lazer right now. Heck if I didn't know better I would think I started cheating :D

I fell in love with osu!lazer. It could be the sound design, visual aesthetics, or user interface improvements, that resonate particularly well with me. I often catch myself just sitting in front of a main menu screen when it goes to that audio visualizer mode, or I watch how the little icons on main menu buttons jump to the beat. It's these small things that made me fall in love with the game.

EDIT: I just managed an easy map with Double Time and Hard Rock which is something I would never imagine just couple of days ago

**Comment by peppy** [comment]
glad to hear you're enjoying the game 😄

**Reply by SolsticeSpectrum** [reply]
You guys are doing really good with lazer. If it has an impact on performance of someone like me who sucks at games, you must be doing something right xd

I mean look at the statistics
![image](https://github.com/ppy/osu/assets/38611351/a7796d58-059d-4a4c-9713-b288f7a5d7b6)

I don't know what kind of secret sauce is causing this but it works :D
